exampleSite/config.toml

```diff
@@ -37,6 +37,10 @@ copyright = "© {year}"
   #   "Twitter", "GitHub", "Email", "Facebook", "GitLab", "Instagram", "LinkedIn", "YouTube"
   iconTitles = ["Twitter", "GitHub"]
 
+  # Metadata for Twitter cards, defaults to params.twitter
+  # twitterSite = "@<your handle>"
+  # twitterAuthor = "@<your handle>"
```

Comment:
If the user doesn't add these values, and it defaults to `params.twitter`, that would result in "improper" meta tags, basically what we had before this PR. Am I understanding correctly? Just trying to wrap my brain around this 😄